src/components/typeahead/typeahead.vue

```diff
@@ -146,12 +148,11 @@
                     this.filteredItems = undefined
                 }
             },
-            guidGenerator() {
-                let S4 = () => (((1+Math.random())*0x10000)|0).toString(16).substring(1)
-                return S4()+S4();
+            guidGenerator(item) {
```

Comment:
What is item? guidGenerator doesn't seem like it should needs a parameter

Reply:
item is the item id is generated for
it does need it now as ids are generated based on filterKeys parameter - this is needed for excluding selected items from dynamic source list